classes/Combination.php

```diff
@@ -71,6 +71,12 @@ class CombinationCore extends ObjectModel
 
     public $available_date = '0000-00-00';
 
+    /** @var string Object creation date in mysql format Y-m-d */
+    public $date_add;
+
+    /** @var string Object last modification date in mysql format Y-m-d */
```

Comment:
`Y-m-d` or `Y-m-d H:i:s` ? because it's a datetime

Reply:
you're right ... I will check ! Thanks

Reply:
done